env0/errors.go

```diff
@@ -11,15 +12,14 @@ import (
 )
 
 func driftDetected(err error) bool {
-	if frerr, ok := err.(*http.FailedResponseError); ok && frerr.NotFound() {
+	var failedResponseError *http.FailedResponseError
```

Comment:
This is the right way to handle errors starting go 1.13.
(Errors in golang can be wrapped within other errors, casting is no longer the correct way).